app/models/payment.rb

```diff
@@ -1,7 +1,7 @@
 class Payment < ApplicationRecord
   belongs_to :billing_contract, optional: true
 
-  validates :amount, presence: true
+  validates :amount, presence: true, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
i dont think equal to 0 is a valid payment amount

Reply:
refactor